activiti-cloud-starter-runtime-bundle/pom.xml

```diff
@@ -38,6 +38,11 @@
       <artifactId>activiti-cloud-services-rest-impl</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.activiti.cloud</groupId>
+      <artifactId>activiti-cloud-services-connectors</artifactId>
+      <version>${project.version}</version>
```

Comment:
@erdemedeiros this is probably wrong, and it means that the new module is missing in the dependencyManagement in the parent pom